gammapy/data/data_store.py

```diff
@@ -284,6 +286,29 @@ def load_many(self, obs_ids, hdu_type=None, hdu_class=None):
 
         return things
 
+    def check_observations(self):
+        """Check all observations regarding their values in events, aeff, edisp, psf
```

Comment:
Say in the docstring what the method does. I.e. if you keep as-is, say that it's logging the results.